utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/sink/UniversalKafkaSinkFactory.scala

```diff
@@ -340,13 +374,24 @@ class UniversalKafkaSinkFactory(
     )
 
     val clientId = s"${TypedNodeDependency[MetaData].extract(dependencies).name}-${preparedTopic.prepared}"
-    val validationMode = if (params.extractUnsafe[Boolean](sinkRawEditorParamName)) {
-      validationModeParamDeclaration.extractValue(params) match {
-        case Some(validationModeString) => extractValidationMode(validationModeString)
-        case None                       => ValidationMode.strict
+    val validationMode = {
+      if (params.isPresent(sinkRawEditorParamName)) {
+        if (params.extractUnsafe[Boolean](sinkRawEditorParamName)) {
+          validationModeParamDeclaration.extractValue(params) match {
+            case Some(validationModeString) => extractValidationMode(validationModeString)
+            case None                       => ValidationMode.strict
+          }
+        } else {
+          ValidationMode.strict
+        }
+      } else if (params.isPresent(sinkValidationModeParamName)) {
+        validationModeParamDeclaration.extractValue(params) match {
+          case Some(validationModeString) => extractValidationMode(validationModeString)
+          case None                       => ValidationMode.strict
```

Comment:
This is a dead code. If you checked `isPresent`, `extractValue` will always return true.